server/models.py

```diff
@@ -16,18 +17,25 @@
 class Scenario(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     name: Mapped[str] = mapped_column(nullable=False)
+    executions: Mapped[List["Execution"]] = relationship(
+        back_populates="scenario")
```

Comment:
Why is this needed? Is this also a column in the DB or just a reference in the ORM-object?